core/src/main/resources/jenkins/security/ApiTokenProperty/help-tokenStore.html

```diff
@@ -0,0 +1,12 @@
+<div>
+    The API tokens can be used for authenticating yourself in the REST API call.
+    See <a href="https://jenkins.io/redirect/api-token">our wiki</a> for more details.<br/>
+    The associated username with the tokens is the regular one of your account.<br/>
+    <br/>
+    The best practices in terms of tokens are:
+    <ul>
+        <li>Use a different token for each application (if one application is compromised, you do not loose data from other)</li>
+        <li>Regenerate the tokens every x weeks/months. We put an indicator concerning the age of the token.</li>
```

Comment:
Maybe mention a specific timeframe (6 months?) instead of just using `x`.

Reply:
Also added `(depending on your context)`